src/collatzConjecture.js

```diff
@@ -0,0 +1,17 @@
+export default function collatzConjecture(n){
```

Comment:
It's good that you got this to pass, but in the future you should look at refactoring this using additional array methods. Also, try to walk through this code with some comments to reinforce the learning.